addons/stock/report/stock_report_views.xml

```diff
@@ -86,40 +86,12 @@
             <field name="report_name">stock.report_stock_rule</field>
             <field name="report_file">stock.report_stock_rule</field>
         </record>
-        <record id="label_product_template" model="ir.actions.report">
-            <field name="name">Product Label (ZPL)</field>
-            <field name="model">product.template</field>
-            <field name="report_type">qweb-text</field>
-            <field name="report_name">stock.label_product_template_view</field>
-            <field name="report_file">stock.label_product_template_view</field>
-            <field name="binding_model_id" ref="product.model_product_template"/>
-            <field name="binding_type">report</field>
-        </record>
         <record id="label_product_product" model="ir.actions.report">
             <field name="name">Product Label (ZPL)</field>
             <field name="model">product.product</field>
             <field name="report_type">qweb-text</field>
             <field name="report_name">stock.label_product_product_view</field>
             <field name="report_file">stock.label_product_product_view</field>
```

Comment:
The `binding_model_id` should also be reset on this record.

Reply:
indeed :+1: